tomni/cytosmart_data_format/utils/main.py

```diff
@@ -9,3 +10,33 @@ def parse_points_to_contour(points: List[Point]) -> np.ndarray:
     contour_points = [[[point.x, point.y]] for point in points]
     contour = np.array(contour_points, dtype=np.int32)
     return contour
+
+
+def compress_polygon_points(
```

Comment:
should be a combine approach not so all polygons with more than 1000 points should use the naive version